plugins/woocommerce-admin/client/products/fills/constants.ts

```diff
@@ -1,17 +1,32 @@
-export const PRICING_SECTION_BASIC_ID = 'pricing/basic';
-export const PRICING_SECTION_TAXES_ID = 'pricing/taxes';
-export const DETAILS_SECTION_ID = 'general/details';
-export const INVENTORY_SECTION_ID = 'inventory/inventory';
-export const INVENTORY_SECTION_ADVANCED_ID = 'inventory/advanced';
-export const IMAGES_SECTION_ID = 'general/images';
-export const ATTRIBUTES_SECTION_ID = 'general/attributes';
-export const SHIPPING_SECTION_BASIC_ID = 'shipping/shipping';
-export const SHIPPING_SECTION_DIMENSIONS_ID = 'shipping/dimensions';
-
-export const TAB_INVENTORY_ID = 'tab/inventory';
 export const TAB_GENERAL_ID = 'tab/general';
-export const TAB_SHIPPING_ID = 'tab/shipping';
 export const TAB_PRICING_ID = 'tab/pricing';
+export const TAB_INVENTORY_ID = 'tab/inventory';
+export const TAB_SHIPPING_ID = 'tab/shipping';
+export const TAB_OPTIONS_ID = 'tab/options';
+
+export const VARIANT_TAB_GENERAL_ID = `variant/${ TAB_GENERAL_ID }`;
```

Comment:
All of the IDs here have been refactored into a sort of breadcrumbs to try to keep them consistent.

Reply:
This does make sense, although I wonder if we should change them to `dashes` or `underscores` at some point.
Mostly because in our fill components we prepend this: `woocommerce_product_tab_`
Turning something like this into: `woocommerce_product_tab_tab/options`.
Given this is purely a string this is not a huge deal by any means, and we could even modify this only within our `slotfill` wrappers.
So maybe forget I said anything :)

Reply:
Good thoughts! The ID structure is a little scattered right now. I suppose it's important to remember that we have two different kinds. We have the Slot ID's that need to be exactly the same for the Slot's and any Fill's that want to go into those slots, but I'm also generating IDs completed unique to each fill for the `SlotContext` stuff. 

We're also using `name` and `id` terminology somewhat interchangeably in the code.

I agree that it's worth giving a little more thought to how these are generated. I actually just realized that the WooProductFieldItem component wasn't generating unique IDs for fills with different sections, so I pushed another commit to fix that. The rest is worth thinking over, but I'll leave that for another PR for now.